utils/markdown.go

```diff
@@ -35,12 +37,19 @@ func StripMarkdown(markdown string) (string, error) {
 
 // MarkdownToHTML takes a string containing Markdown and returns a string with HTML tagged version
 func MarkdownToHTML(markdown string) (string, error) {
+	re := regexp.MustCompile(`<(.*)>.*?|<(.*) />`)
```

Comment:
This regex isn't going to work unfortunately. Obviously we can't use `html.EscapeString` on the whole thing because of the issue with block quotes, but I'd be worried we'd miss something with anything like this that's much more complicated than just replacing the unsafe characters with HTML entities. Generally, trying to use regexes to do much with HTML is a bad idea.

Reply:
Hi @hmhealey, how about doing the opposite? 
I've re-added the html.EscapeString step but within utils.MarkdownToHTML I now search for > at the beginning of a line to then unescape just that for blockquote parsing?